<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>21</number>
    </property>
    <property name="topMargin">
     <number>21</number>
    </property>
    <property name="rightMargin">
     <number>21</number>
    </property>
    <property name="bottomMargin">
     <number>21</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="LSystemView" name="lsystemView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>281</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>Display</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Depth</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="depthSpinBox">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="boldLines">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Bold lines</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="autoTreeColors">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Correct trunk color</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>281</width>
          <height>311</height>
         </size>
        </property>
        <property name="title">
         <string>Rules</string>
        </property>
        <widget class="QLineEdit" name="ruleF">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>F →</string>
         </property>
        </widget>
        <widget class="QPushButton" name="submit">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>270</y>
           <width>131</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Submit / Regen</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="axiom">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>60</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>59</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Axiom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>X →</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ruleX">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>120</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Y →</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ruleY">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>150</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>A →</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ruleA">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>180</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>B →</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ruleB">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>210</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Constants</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="constF">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>F</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="constX">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="constY">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="constA">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="constB">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>30</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="angle">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>240</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>359</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Angle</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LSystemView</class>
   <extends>QWidget</extends>
   <header location="global">lsystemview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
